build-system/pr-check/e2e-tests.js

```diff
@@ -28,6 +28,7 @@ const {
   startTimer,
   stopTimer,
   timedExecOrDie: timedExecOrDieBase,
+  timedExecOrDieWithDnsMonitor: timedExecOrDieWithDnsMonitorBase,
```

Comment:
With my suggestion to change how `gulp dns-monitor` works, this file can be reverted.